paddle/fluid/inference/api/paddle_api.h

```diff
@@ -123,7 +123,8 @@ class ZeroCopyTensor {
    */
   template <typename T>
   T* mutable_data(PaddlePlace place);
-  /** Get the memory directly, will return the place and memory size by pointer.
+  /** Get the memory directly, will return the place and element size by
```

Comment:
The size is not the memory size, according to https://github.com/PaddlePaddle/Paddle/blob/fb63cd89d4343270ad96598aac177a7ad8d36c21/paddle/fluid/inference/api/details/zero_copy_tensor.cc#L54-L67

So change https://github.com/PaddlePaddle/Paddle/pull/15250/files#diff-419bceacde82e0efd8c87a54e8f110f1R357